bin/benchpark

```diff
@@ -86,10 +116,14 @@ def benchpark_setup_handler(args):
     benchmark = args.benchmark
     system = args.system
     workspace_root = pathlib.Path(os.path.abspath(args.workspace_root))
-
     source_dir = source_location()
-
     debug_print(source_dir)
+    debug_print(benchmark)
```

Comment:
I recommend providing more context in the message (I'm guilty of not doing that in the original PR where I did this).